src/licensedcode/data/licenses/utopia.yml

```diff
@@ -0,0 +1,8 @@
+key: utopia
+short_name: Utopia Typeface License 
+name: License to TeX Users Group for the Utopia Typeface
+category: Permissive
```

Comment:
@DennisClark this looks permissive alright to me, do you agree?

Reply:
@DennisClark ping here too

Reply:
Permissive is fine here.